components/map/map.tsx

```diff
@@ -22,26 +22,26 @@ const DeckGLOverlay = props => {
   return null;
 };
 
-const onClick = info => {
-  if (info.object) {
-    // eslint-disable-next-line
-    alert(`${info.object.properties.name} (${info.object.properties.abbrev})`);
-  }
-};
-
 type MapProps = {
   bikeShareData: BikeShareFeature[];
   amtrakData: AmtrakResponse;
 };
 
 const Map: React.FC<MapProps> = ({ bikeShareData, amtrakData }) => {
+  const getTooltip = useCallback(
```

Comment:
I know this wasn't on the list but it seems like enough of a common use case that it was worth adding. also didn't know what else to do at the end of the day.

Reply:
looks great!